provider/src/main/kotlin/org/gradle/kotlin/dsl/accessors/AccessorsClassPath.kt

```diff
@@ -115,40 +100,36 @@ fun buildAccessorsJarFor(projectSchema: ProjectSchema<String>, classPath: ClassP
     })
 }
 
-
 internal
 sealed class TypeAccessibility {
     data class Accessible(val type: String) : TypeAccessibility()
     data class Inaccessible(val type: String, val reasons: List<InaccessibilityReason>) : TypeAccessibility()
 }
 
-
 internal
 sealed class InaccessibilityReason {
 
     data class NonPublic(val type: String) : InaccessibilityReason()
     data class NonAvailable(val type: String) : InaccessibilityReason()
     data class Synthetic(val type: String) : InaccessibilityReason()
 
-    val explanation get() = when (this) {
```

Comment:
I would also call this a bug. When only a `get` function is supplied for a property this shorthand should be preferred.

Reply:
This may just be that there isn't a check for this. I ran the intellij formatter to get rid of most of the problems with star imports that Ktlint can't fix for you. Both code patterns are valid according to the formatter.

The way the tool was designed was such that it doesn't make decisions about format for things that haven't been decided upon by the community here: https://github.com/yole/kotlin-style-guide

Let me revert this format change. Sorry about that.